AppController/lib/infrastructure_manager_client.rb

```diff
@@ -167,6 +167,7 @@ def terminate_instances(options, instance_ids)
     end
     parameters['instance_ids'] = instance_ids
     parameters['region'] = options['region']
+    parameters['IS_VERBOSE'] = options['verbose']
```

Comment:
So, the tools agents somehow set the verbose parameter in uppercase. I realized that while testing this PR the terminate call is missing this parameter. But I intend to fix the upper case parameter in a separate tools PR if that sounds good?